settings.php

```diff
@@ -863,6 +871,7 @@ function get_FTL_data($arg)
 							<label for="zip_file">File input</label>
 							<input type="file" name="zip_file" id="zip_file">
 							<p class="help-block">Upload only Pi-hole backup files.</p>
+							<input type="hidden" name="token" value="<?php echo $token ?>">
```

Comment:
It's under the same `<form>` tag, so I think it does send the token. I added the CSRF check to teleporter and relocated the token just to be sure.